src/scss/components/checkbox/mixins/_hxCheckboxFacade.scss

```diff
@@ -0,0 +1,63 @@
+@import "vars";
```

Comment:
Remove import. "vars" is already imported via `_config.scss`

Reply:
Ok, `@import` statement removed.